ngraph/python/src/ngraph/opset8/ops.py

```diff
@@ -253,3 +253,25 @@ def matrix_nms(
     }
 
     return _get_node_factory_opset8().create("MatrixNms", inputs, attributes)
+
+
+@nameable_op
+def gather(
+        data: NodeInput,
+        indices: NodeInput,
+        axis: NodeInput,
+        batch_dims: Optional[int] = 0,
+) -> Node:
+    """Return a node which performs Gather with support of negative indices.
```

Comment:
please elaborate more about what `support of negative indices` mean.

Reply:
concretized about negative indices